src/lib/styles.js

```diff
@@ -5,16 +5,21 @@ import values from './values';
 
 type BarProps = {
   barColor: string,
+  barHeight: number,
 };
 
+type TabProps = {
+  barHeight: number,
+}
+
 const Bar = styled.View`
   background-color: ${(props: BarProps) => props.barColor};
-  height: ${values.barHeight};
+  height: ${(props: BarProps) => props.barHeight || values.barHeight};
 `;
 
 const TabTrack = styled.View`
   flex-direction: row;
-  height: 46;
+  height: ${(props: BarProps) => (props.barHeight || values.barHeight) - 2};
```

Comment:
It's better if you use defaultProps on MaterialTabs to set the default `props.barHeight` as `values.barHeight`. That way we'd avoid having to check this multiple times